src/areaSearch/areaSearchLandingPage.tsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import { Container } from 'react-grid-system';
+import { useTranslation } from 'react-i18next';
+import { Button } from 'hds-react';
+import { useNavigate } from 'react-router-dom';
+
+import BoxGrid from '../boxGrid/boxGrid';
+import BoxGridBox from '../boxGrid/boxGridBox';
+import { AppRoutes, getRouteById } from '../root/routes';
+
+const AreaSearchLandingPage = (): JSX.Element => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  const ReadMoreButton = ({ target }: { target: string }): JSX.Element => (
+    <Button
+      role="link"
+      variant="secondary"
+      theme="black"
+      onClick={() => window.open(target, '_blank')}
+    >
+      {t('areaSearch.landingPage.readMore', 'Read more')}
+    </Button>
+  );
+
+  return (
+    <div className="AreaSearchLandingPage">
+      <Container>
```

Comment:
Not 100% sure, but I guess the <Container className="AreaSearchLandingPage"> could be the parent tag here. Isn't it just a normal div with some bootstrap css classes injected in it?

Imo when ever possible to "save" one step of intendation in react JSX, the code is much readable.